### ISSUE TITLE
Narrative buttons not properly designed (WCAG 4.1.2) [QAC]

### ISSUE THREAD

**Issue by ftourangeau** [MeD-DMC/Adapt-accessibilityfixes#204]
**COURSE:**
https://esdc-edsc-adapt.canadacentral.cloudapp.azure.com/preview/5dc446fd27f08836496b6b9e/63a32079b75e454dafed1e1d/
*******************************************************************************
**Page:**
Lesson 2: Domestic Print
Lesson 3: Observation Print
Lesson 4: Manual QA

**Description:**
Under "Overview of the Observation Print Module" there is a group of elements that navigate the slides. But they are not properly designed in terms of accessibility:
   • Navigation buttons do not have accessible names
   • Accessibility features have not been implemented

The element should have a proper name and role to provide information to the user as to their purpose and which is currently active.

Please
   • Mark up elements in a way that reflects their structure and appropriately label them.

Please see the attached screenshot(s) for example.
![image](https://user-images.githubusercontent.com/93537804/229620640-5e73a523-f5d9-491d-8684-4db7a4aac476.png)
